doctor/docs/base.py

```diff
@@ -238,7 +238,7 @@ def get_json_lines(annotation: ResourceAnnotation, field: str, route: str,
         else:
             return []
     else:
-        parameters = annotation.logic._doctor_signature.parameters
+        parameters = annotation.annotated_parameters
         properties = {k: p.annotation for k, p in parameters.items()}
```

Comment:
Using the `annotated_parameters` attribute fixes docs generation as it only includes params whose annotation is a subclass of `doctor.types.SuperType`